packages/app/cypress.config.ts

```diff
@@ -6,7 +6,7 @@ const CYPRESS_INTERNAL_CLOUD_ENV = getenv('CYPRESS_INTERNAL_CLOUD_ENV', process.
 const CYPRESS_INTERNAL_DEV_PROJECT_ID = getenv('CYPRESS_INTERNAL_DEV_PROJECT_ID', process.env.CYPRESS_INTERNAL_DEV_PROJECT_ID || 'sehy69')
 
 export default defineConfig({
-  projectId: CYPRESS_INTERNAL_CLOUD_ENV === 'staging' ? 'ypt4pf' : CYPRESS_INTERNAL_DEV_PROJECT_ID,
+  projectId: CYPRESS_INTERNAL_CLOUD_ENV === 'staging' || CYPRESS_INTERNAL_CLOUD_ENV === 'production' ? 'ypt4pf' : CYPRESS_INTERNAL_DEV_PROJECT_ID,
```

Comment:
Our project ID is the same for staging and production, so we should use the same ID for both envs